### DISCUSSION TITLE
How do I change the remote of gin sibling?

### DISCUSSION THREAD

**Discussion by camiscaff** [LAAC-LSCP/ChildProject#340] [Q&A]
Current remote is  [git@gin.g-node.org:/LAAC-LSCP/tsimane2018.git (git)]
we want to be : [git@gin.g-node.org:/LAAC-LSCP/tsimanec2018.git (git)] #notice the addition of the "c"

**Comment by camiscaff** [comment]
Another question: (ChildProjectVenv) (base) cscaff@MN058AD42G86 tsimanem2018 % datalad push --to gin
[ERROR  ] Unknown push target 'gin'. Known targets: 'origin'. [publish(/Users/cscaff/tsimanem2018)] 
publish(error): . [Unknown push target 'gin'. Known targets: 'origin'.]
(ChildProjectVenv) (base) cscaff@MN058AD42G86 tsimanem2018 % datalad push         
publish(ok): . (dataset) [refs/heads/master->origin:refs/heads/master [new branch]]                                         
(ChildProjectVenv) (base) cscaff@MN058AD42G86 tsimanem2018 % datalad siblings
.: here(?) [git]
.: origin(?) [git@gin.g-node.org:/LAAC-LSCP/tsimanem2018.git (git)]

How can I harmonize the behavior of the two folders tsimanem2018 and tsimanec2018 so I can push identically ( with "push --to gin" or simply "push")

**Comment by lucasgautheron** [comment]
To configure the "origin" sibling to point towards `<newurl>`:

```bash
datalad siblings configure -s origin --url <newurl>
```
If the sibling does not exist, the command will create it.

By default, when cloning/installing a dataset from a source, the sibling is automatically named `origin` and it is configured as the default sibling. Therefore I would recommend always configuring the main sibling with the name "origin" always rather than "gin" locally.

More generally, I strongly advise using these procedures to create new LAAC or EL1000 datasets: https://github.com/LAAC-LSCP/datalad-procedures

These procedures automatically configure the siblings and the structure of the dataset.

More datalad documentation here: http://docs.datalad.org/en/stable/generated/man/datalad-siblings.html

**Comment by alecristia** [comment]
thank you! We had indeed forgotten about procedures. So now the situation is that we have a local version that was not created using procedures, and which has been pushed to GIN. There is not a lot of work on it, so we can re-start from scratch.

Would you suggest that? And if so, would it really be from scratch: force remove local copy, delete copy on GIN, and start over?

Thanks in advance!

**Reply by lucasgautheron** [reply]
I would recommend that. It's usually easier to work from a clean state, unless you feel very confident you can tweak your way back to that state!

I would suggest to:
 1. Remove the datasets from GIN (delete them from GIN's interface) 
 2. Keep a copy of your files
 3. Recreate a dataset with these procedures
 4. Fill these new datasets with the backup of your past progress

**Reply by alecristia** [reply]
5. Have a pint

**Reply by lucasgautheron** [reply]
Yes (preferably in that order)
